doc/changelog/11-infrastructure-and-dependencies/11358-410.rst

```diff
@@ -0,0 +1,4 @@
+- **Added:**
+  Add support for OCaml 4.10
+  (`#11358 <https://github.com/coq/coq/pull/11358>`_,
+  by Kate Deplaix).
```

Comment:
We usually declare an OCaml version "supported" when the CI setup is bumped, for 4.10 see #11131 ; I'd suggest to rephrase the changelog then to reflect the content of the actual patch. Note tho the observation w.r.t. #11102 , I'd suggest you sync with people there before doing more work here.